concepts/eventing-code-guidelines.md

```diff
@@ -389,6 +389,64 @@ MarkReady(apiRuleNew) // instead consider using another function that explicitly
 ```
 </details>
 
+### Logging Rules
+
+Here are some general rules to keep the logs standardized across the eventing components:
+
+- Log message should be as short and meaningful as possible
+- Logs should be aligned with the unified way of logging inside kyma ([reference](https://github.com/kyma-project/community/blob/16520807578a889f763bb8083601ae0de357aa3d/concepts/observability-consistent-logging/README.md))
+- Each log message should have enough context to convey what happened
+- Each log message should have the proper log level ([reference](https://github.com/kyma-project/community/blob/16520807578a889f763bb8083601ae0de357aa3d/concepts/observability-consistent-logging/unified-approach-to-logging-levels.md))
```

Comment:
```suggestion
- Each log message should have the proper log level ([reference](https://github.com/kyma-project/community/blob/main/concepts/observability-consistent-logging/unified-approach-to-logging-levels.md)).
```

lets use a link to main branch instead of specific commit.

Reply:
Done